src/main/atm2lndType.F90

```diff
@@ -89,6 +89,10 @@ module atm2lndType
      real(r8), pointer :: forc_po2_240_patch            (:)   => null() ! 10-day mean O2 partial pressure (Pa)
      real(r8), pointer :: forc_aer_grc                  (:,:) => null() ! aerosol deposition array
      real(r8), pointer :: forc_pch4_grc                 (:)   => null() ! CH4 partial pressure (Pa)
+     real(r8), pointer :: forc_rainc_grc                (:)   => null() ! convective rain (mm/s)
+     real(r8), pointer :: forc_rainl_grc                (:)   => null() ! large scale rain (mm/s)
+     real(r8), pointer :: forc_snowc_grc                (:)   => null() ! convective snow (mm/s)
+     real(r8), pointer :: forc_snowl_grc                (:)   => null() ! large scale snow (mm/s)
```

Comment:
> I feel it would be best if we avoided adding these variables to atm2lndType. Since they are only needed in the scope of lnd_import, I think they can be local arrays in that routine (declared like `forc_rainc(bounds%begg:bounds%endg)`, as in the prior version of the lilac and nuopc caps), then passed individually to your new shared subroutine. (i.e., the new subroutine would have 4 extra arguments, accepting these 4 arrays). The point of this would be to avoid bloating this derived type. But I don't feel strongly about this.

I made the change and submitted this test:
`./create_test ERS_D_Ld6.f10_f10_musgs.I1850Clm45BgcCrop.cheyenne_intel.clm-clm50CMIP6frc -c /glade/p/cgd/tss/ctsm_baselines/ctsm1.0.105_qsat` **PASS**

Reply:
I forgot to remove these from atm2lndType, so doing that now.